lib/Core/Executor.cpp

```diff
@@ -828,6 +832,7 @@ void Executor::branch(ExecutionState &state,
       es->ptreeNode->data = 0;
       std::pair<PTree::Node*,PTree::Node*> res = 
         processTree->split(es->ptreeNode, ns, es);
+      if (DumpProcessTree) dumpPTreeCSV(es->ptreeNode);
```

Comment:
I think you should put the body of if on a new line. Also maybe run clang-format.